system-tests/e2e-transfer-test/runner/src/test/java/org/eclipse/edc/test/e2e/TransferEndToEndTestBase.java

```diff
@@ -71,9 +71,9 @@ protected void createResourcesOnProvider(String assetId, Map<String, Object> dat
         PROVIDER.createContractDefinition(assetId, UUID.randomUUID().toString(), noConstraintPolicyId, noConstraintPolicyId);
     }
 
-    protected void awaitTransferToBeInState(String transferProcessId, TransferProcessStates state) {
+    protected void awaitTransferToBeInState(TransferEndToEndParticipant participant, String transferProcessId, TransferProcessStates state) {
```

Comment:
I would put this method in the `Participant` class, so it will be directly available for every participant instance